crypto/evp/pmeth_lib.c

```diff
@@ -1070,6 +1313,8 @@ static int legacy_ctrl_str_to_param(EVP_PKEY_CTX *ctx, const char *name,
     else if (strcmp(name, "ecdh_kdf_md") == 0)
         name = OSSL_EXCHANGE_PARAM_KDF_DIGEST;
 # endif
+    else if (strcmp(name, "N") == 0)
```

Comment:
Why just this one??

Reply:
Its the only one where the parameter name is different to the ctrl string name.